cmd/yggdrasil/main.go

```diff
@@ -292,10 +292,10 @@ exit:
 }
 
 func (n *node) shutdown() {
-	n.core.Stop()
 	n.admin.Stop()
 	n.multicast.Stop()
 	n.tuntap.Stop()
+	n.core.Stop()
```

Comment:
Order seems reasonable to me... as long as the core can still try to write to the tun/tap, and the tun/tap just fails when this happens.